### PR TITLE
Add sensor screen tests

### PR DESCRIPTION
**Pull request by xiuqijix**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25692)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/351215214)

File: sensors/generic-sensor-tests.js [hunk 1]
```diff
@@ -242,6 +250,24 @@ function runGenericSensorTests(sensorType) {
...
+    return;
+  }
+
+  promise_test(async t => {
+    const sensor = new sensorType({referenceFrame: "screen"});
+    const sensorWatcher = new EventWatcher(t, sensor, ["reading", "error"]);
+    sensor.start();
+
+    await sensorWatcher.wait_for("reading");
+    //TODO No way to test sensor APIs
```
Comment (Honry) [inline sensors/generic-sensor-tests.js]: Can you explain the detailed TODO thing in the annotation?

Comment (kaixinjxq) [reply sensors/generic-sensor-tests.js]: Updated the annotation.

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Pull request merged**